packages/@angular/cli/blueprints/class/index.ts

```diff
@@ -11,11 +13,18 @@ export default Blueprint.extend({
       name: 'spec',
       type: Boolean,
       description: 'Specifies if a spec file is generated.'
+    },
+    {
+      name: 'app',
```

Comment:
This gets repeated a lot. It's probably better to make some kind of base `BaseAppOptions`+`AppOptions` similar to `baseBuildCommandOptions` (or a better abstraction if you can find it).

That way if we need to change it, or add more app options in the future, it's only in one place.